tests/test_threads.py

```diff
@@ -180,3 +181,16 @@ def test_should_abort():
 
         should_abort()
         ThreadPool().query_abort.called_once_with()
+
+
+def test_stop_threads():
+    with patch("guikit.threads.ThreadPool", MagicMock()):
+        from guikit.threads import ThreadPool
+
+        _COUNT = 3
+        for _ in range(_COUNT):
+            ThreadPool().run_thread(lambda: threading.sleep(0.5))
```

Comment:
If you use the real `ThreadPool` this will fail unless you provide an actual `window` as input. Check the other tests that do that: you should use the `window` fixture already in place.